src/homekit.cs

```diff
@@ -1838,6 +1838,10 @@ interface HMCameraSnapshotControlDelegate
 	{
 		[Export ("cameraSnapshotControl:didTakeSnapshot:error:")]
 		void DidTakeSnapshot (HMCameraSnapshotControl cameraSnapshotControl, [NullAllowed] HMCameraSnapshot snapshot, [NullAllowed] NSError error);
+
+		// Added in iOS 10.1 SDK but it is supposed to be present in iOS 10.
```

Comment:
Let's not assume the *lack* of the macros in the headers means it's available in earlier iOS versions. It's a slippery road and we cannot really detect such cases in protocols (like a delegate) since we can't be sure about optionals.

Also the code above will add this API to tvOS 10 - which is not in the headers and would be a breaking change to remove later, so `[iOS (10,1)][Watch (3,1)][NoTV]` is safer.